### ISSUE TITLE
Screen Reader is not announcing selected/non-selected state in a DetailsList, via pressing spacebar

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7436]
### Environment Information 
Test Environment:
OS version: Version 1803 (OS build 17134. 228)
Browser: Edge, Chrome
Screen Reader: Narrator, JAWS
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 Pre-Requisite:
1. Turn on narrator by pressing Ctrl + window + ENTER combination.

Repro Steps:
1. Open https://preview.web.powerapps.com/  and login using valid credentials.
2. Navigate to Data Drop down on left pane and select it.
3. Navigate to Entities option in the drop down list and select it.
4. Page with all the entities appears, navigate to "Export Data" menu item on header and select it.
5. Page with entities opens, navigate to the entities and select it.
6. Navigate with the Narrator and reach "Account" cell.
7. Verify whether after pressing spacebar Narrator announce select/unselected state or not.

 
#### Actual behavior:
 When user navigate over the table with the narrator and reach 'Account' and press spacebar then the narrator does not announce select/unselected state. Instead Narrator announce "Space".

 #### Expected behavior:
 
Whenever, user navigates over the table with the narrator and reach 'Account' and press spacebar then the narrator should announce select/unselected state. 

**Note:**
1. This issue also exist with Chrome + JAWS.  When user press spacebar then the JAWS announce out of table.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low)  Blocking 
![mas1 3 1-narrator does not announce information of selected](https://user-images.githubusercontent.com/30456910/50204922-6a958780-038c-11e9-9f3a-dcacb601c24c.png)

 
Products/sites affected: (if applicable)

**Comment by KevinTCoughlin** [comment]
Duplicate of https://github.com/OfficeDev/office-ui-fabric-react/issues/6525#issuecomment-430333405

**Comment by ecraig12345** [comment]
Closing this in favor of #6525 since it's the same root cause and that one has a lot more investigation and discussion.
